index.html

```diff
@@ -31,13 +32,209 @@ <h2 id="job-title">I am a Software Developer</h2>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <li><a href="#"><button class="project-1-buttons">html</button></a></li>
+                            <li><a href="#"><button class="project-1-buttons" >bootstrap</button></a></li>
+                            <li><a href="#"><button class="project-1-buttons">Ruby</button></a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1 last-project">
+                    <img src="images/Img Placeholder-2.png" alt="printed-magazine">
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">html</button></a></li>
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">bootstrap</button></a></li>
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">Ruby</button></a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+            </section>
+            <section class="about-me">
+                <div class="about-div">
+                    <h2 class="about-me-title">About Me</h2>
+                </div>
+                <div class="about-me-description-div">
+                    <p class="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#"><button class="get-resume-button">Get My Resume</button></a>
+                    <div class="side-image-div"><img class="side-image" src="images/about-me-side-image.png" alt=""></div>
```

Comment:
- [ ] You did well inserting the background image as requested by the previous reviewer but to follow the best practices as it is one major requirement of this milestone, kindly make sure to insert the background image using the CSS `background` properties which is the professional recommended way to do so. :wink: [check this](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) for more info.